docs-kits/kits/Eco_Pass_KIT/page-software-development-view.md

```diff
@@ -11,43 +13,263 @@ Thereby, this KIT covers various aspects, starting from how the available API En
+| token      | searchSessionToken     | [REQUIRED]                                                                      |
+
+##### /api/contract/decline                                                   
+
+| Parameter  | Value Name             | Mandatory or Optional Value |
+|------------|------------------------|-----------------------------|
+| processId  | processIdentification  | [REQUIRED]                  |
+| token      | searchSessionToken     | [REQUIRED]                  |
+
+
+##### /api/contract/cancel
+
+| Parameter  | Value Name             | Mandatory or Optional Value |
+|------------|------------------------|-----------------------------|
+| processId  | processIdentification  | [REQUIRED]                  |
+| contractId | contractIdentification | [REQUIRED]                  |
+| token      | searchSessionToken     | [REQUIRED]                  |
+
+
+
+##### /api/contract/status/{processId}   
+| Parameter | Value Name            | Mandatory or Optional Value |
+|-----------|-----------------------|-----------------------------|
+| processId | processIdentification | [REQUIRED]                  |
+                                                                                      
+
+##### /api/data 
+
+| Parameter  | Value Name             | Mandatory or Optional Value |
+|------------|------------------------|-----------------------------|
+| processId  | processIdentification  | [REQUIRED]                  |
+| contractId | contractIdentification | [REQUIRED]                  |
+| token      | searchSessionToken     | [REQUIRED]                  |
+                                                                                                                                                                                
+
+#### External API calls
+
+| Service   | Method | Path                                               | Params                                     |
+| ------ | ------ | -------------------------------------------------- | ------------------------------------------ |
+| Discovery Service [CX-0053] | POST   | /api/administration/connectors/discovery/search    | key:manufacturerPartId (BPN Discovery) & key:bpn (EDC Discovery)                          |
+| BPN Discovery Service [CX-0053]  | POST   | /api/administration/connectors/bpnDiscovery/search | key:manufacturerPartId                          |
+| EDC Discovery Service [CX-0001]  | POST   | /api/administration/connectors/discovery/search    | `{[<Company's BPNL>]}`                     |
+| Digital Twin Registry [CX-0002]  | GET    | /lookup/shells                                     | key:partInstanceID, value: batteryDMC-Code or Product Identification Code |
+| Digital Twin Registry [CX-0002]  | GET    | /shell-descriptors/{digitalTwinId}                                     | key:partInstanceID, value: batteryDMC-Code or Product Identification Code |
+
+
+## Reference Implementations
```

Comment:
If you have an API as reference implementation please add this to the KIT via OpenAPI Plugin e.g. : https://eclipse-tractusx.github.io/docs-kits/kits/Business%20Partner%20Kit/Software%20Development%20View/Pool%20Api/get-sites-paginated

Reply:
The API plugin will no be anymore supported.

We have referenced our OpenApi specification here: 

![image](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/71728767/6dc8dda1-f5c2-4f0d-887e-77b368430e41)

https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/631/files/2679879bf3729d525fb9d43a660e51921b0aa823#diff-c8b6ae86bef984ee00e1927ccc4407795adcb2b5757dd0c799d71a23d160bd99R184

In this matrix chat you can confirm my afirmation: https://chat.eclipse.org/#/room/!oXNwXGsvkbDUMiQhms:matrix.eclipse.org/$W63RrxowCqp9qmwF4QkfcjumeyhZ1dWIo2udbbrWyAk?via=matrix.eclipse.org&via=matrix.org&via=beeper.com


It was stated by @SebastianBezold here: https://chat.eclipse.org/#/room/!oXNwXGsvkbDUMiQhms:matrix.eclipse.org/$jRRigM3k-kVYdLhY0K4eHfPFhDqFaOLDKVtRQedUQyY?via=matrix.eclipse.org&via=matrix.org&via=beeper.com

Reply:
We are still maintaining the plugin but if you are using Swaggerhub, better, thanks.